File: SEIST/polyglot/vb/SeisReleasePolicy.vb
Public Structure SeisReleasePolicy
    Public ReducedMotionRequired As Boolean
    Public DependencyBudgetLocked As Boolean
    Public LiveUploadRequiresTarget As Boolean
End Structure

Module SeisPolicyModule
    Public ReadOnly DefaultPolicy As New SeisReleasePolicy With {
        .ReducedMotionRequired = True,
        .DependencyBudgetLocked = True,
        .LiveUploadRequiresTarget = True
    }
End Module

File: SEIST/polyglot/scala/SeisReleasePolicy.scala
final case class SeisReleasePolicy(
  reducedMotionRequired: Boolean,
  dependencyBudgetLocked: Boolean,
  liveUploadRequiresTarget: Boolean
)

object SeisReleasePolicy {
  val default: SeisReleasePolicy = SeisReleasePolicy(
    reducedMotionRequired = true,
    dependencyBudgetLocked = true,
    liveUploadRequiresTarget = true
  )
}
